docs/admin_guide/variables.rst

```diff
@@ -0,0 +1,17 @@
+.. _hopsworks-administration:
+
+==================
+Hopsworks settings
+==================
+
+Clicking on the *Edit variables* icon on the admin panel, will lead to the configuration management panel.
+From here administrators are able to change the configuration and behavior of Hopsworks.
+
+Hopsworks UI doesn't have a configuration file, instead the configuration is stored in the database in the `hopsworks.variables` table. 
+
+This panel in the admin UI allows easy access to that configuration. Administrators are able to view and edit it from the panel.
+Administrators can edit the value of a variable by clickin on the pencil. After the edit is done, administrators should click on `Reload variables` to instruct Hopsworks to invalidate its configuration cache and reload it from the database.
```

Comment:
The new configuration will not take affect until the the `Reload variables` button has been clicked.